scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -1311,7 +1322,8 @@ Else
     $BypassInstallationFlag = $null
 }
 
-$retCode = RunTest
+# Evaluate collection of test results for final pass/fail result
+RunTest | ForEach-Object {$retCode = 0} {$retCode = $retCode -bor $_}
 Write-Host "Exit test with code $retCode"
 
-Exit $retCode -gt 0
\ No newline at end of file
+Exit $retCode
```

Comment:
@philipktlin @damonbarry is it safe to change this to return the return code directly? I noticed that the -gt condition means that the script will return 1 if the tests all pass.

Reply:
When a call to RunLeafDeviceTest succeeds, it returns 0. When you OR the results together the final result will also be 0 (if they all succeed). So the script will return the value of `0 -gt 0`, which is $False, which converts to 0, not 1.

Reply:
it should be safe for this change.  However $retCode should be 0 if all tests passed.  If $retCode is greater than 0, ($retCode -gt 0) is equal to $True, which is equivalent to 1.  Exit with value 1 means test is failed.

Reply:
Ahh my mistake, it was working as intended. I was commenting the failing tests in and out to test the output. I can revert the change to preserve it.